software-modules/programming/sublime/sublime.sh

```diff
@@ -15,12 +15,18 @@
 #		Enya Quetzalli <equetzal@huronos.org>
 
 set -xe
-NAME=sublime
+NAME=sublime-text
```

Comment:
Be aware that this name must match this files
- https://github.com/equetzal/huronOS-build-tools/blob/development/base-system/usrroot/etc/hmm/all
- https://github.com/equetzal/huronOS-build-tools/blob/development/base-system/usrroot/etc/hmm/any
- https://github.com/equetzal/huronOS-build-tools/blob/development/base-system/usrroot/etc/hsync/all_software

And this same name is the one to be used by the directives file, so you would also be needing to update the documentation to match this nameID on 
https://github.com/equetzal/huronOS-build-tools/blob/development/docs/usage/directives/configurations/software-modules.md

Then you have 3 options:
- Hardcode the 'sublime-text' on the line 29 for the apt command.
- Hardcode 'sublime.hsm' on the save changes and all below names for the module
- Ooor update all the other files to match the new name

Reply:
I'll take the shorcut and hardcode `sublime-text` like the firefox file does.